app/routers/event.py

```diff
@@ -87,16 +93,32 @@ async def create_new_event(request: Request,
 async def eventview(request: Request, event_id: int,
                     db: Session = Depends(get_db)) -> Response:
     event, comments, end_format = get_event_data(db, event_id)
+    start_format = START_FORMAT
+    if event.all_day is True:
+        start_format = '%A, %d/%m/%Y'
+        end_format = ""
     messages = request.query_params.get('messages', '').split("---")
     return templates.TemplateResponse("event/eventview.html",
                                       {"request": request,
                                        "event": event,
                                        "comments": comments,
-                                       "start_format": START_FORMAT,
+                                       "start_format": start_format,
                                        "end_format": end_format,
                                        "messages": messages})
 
 
+UPDATE_EVENTS_FIELDS = {
+    'title': str,
+    'start': datetime,
+    'end': datetime,
+    'availability': bool,
+    'all_day': bool,
+    'content': (str, type(None)),
+    'location': (str, type(None)),
+    'category_id': (int, type(None))
+}
+
+
```

Comment:
Is this part needed?
If not - delete, if it is - please move to more proper location (if it's pydantic's annotations, there is a proper file for that)